app/models.py

```diff
@@ -600,6 +600,20 @@ def get_simple_dict(self):
         }
 
     def get_dict(self):
+        full_address = ''
```

Comment:
could we move this portion of code into its own function? you can see examples on the Spot class

```
    def get_url(self):
        return Spot.create_url(self.id, self.name)

    def get_beach_name_for_url(self):
        return demicrosoft(self.name).lower()

    @classmethod
    def create_url(cls, id, name):
        return '/Beach/'+str(id)+'/'+demicrosoft(name).lower()   
```